### PR TITLE
[Docs] document that `.child()` call triggers `level-change` event.

### PR DESCRIPTION
**Pull request by laSinteZ**
As discussed [in the issue 1006](https://github.com/pinojs/pino/issues/1006#issuecomment-817205040) I've created an update to docs. Unfortunately, I have no bandwidth to fix this bug right now, but I guess letting people know about it is also good.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -840,6 +840,20 @@ logger.on('level-change', (lvl, val, prevLvl, prevVal) => {
 logger.level = 'trace' // trigger event
 ```
 
+Please note that due to a [known bug](https://github.com/pinojs/pino/issues/1006), every `logger.child()` call will
+fire a `level-change` event, with `levelLabel` equal to `previousLevelLabel` and `levelValue` equal to `previousLevelValue`.
+You might want to ignore these events, using something like:
+
+```js
+const logger = require('pino')()
+logger.on('level-change', (lvl, val, prevLvl, prevVal) => {
```
Comment (mcollina) [inline docs/api.md:849]: I would remove the arrow function and rely on `this === logger` to check.

Comment (laSinteZ) [reply docs/api.md]: Thank you, I did not think of this (no pun intended) :)

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by laSinteZ** [comment]
@mcollina pardon me for one more commit, but my statement wasn't correct:
for example, `logger.child({ level: "trace" });` call would result in an event with lvl = trace, prevLvl = info

File: docs/api.md [hunk 2]
```diff
@@ -840,6 +840,20 @@ logger.on('level-change', (lvl, val, prevLvl, prevVal) => {
 logger.level = 'trace' // trigger event
 ```
 
+Please note that due to a [known bug](https://github.com/pinojs/pino/issues/1006), every `logger.child()` call will
+fire a `level-change` event. You might want to ignore these events, using something like:
```
Comment (jsumners) [inline docs/api.md:844]:
```suggestion
Please note that due to a [known bug](https://github.com/pinojs/pino/issues/1006), every `logger.child()` call will
fire a `level-change` event. These events can be ignored by writing an event handler like:
```

File: docs/api.md [hunk 3]
```diff
@@ -840,6 +840,20 @@ logger.on('level-change', (lvl, val, prevLvl, prevVal) => {
...
+
+```js
+const logger = require('pino')()
+logger.on('level-change', function (lvl, val, prevLvl, prevVal) {
+  if (logger === this) {
+    console.log('%s (%d) was changed to %s (%d)', prevLvl, prevVal, lvl, val)
+  }
```
Comment (jsumners) [inline docs/api.md:851]:
```suggestion
  if (logger !== this) {
    return
  }
  console.log('%s (%d) was changed to %s (%d)', prevLvl, prevVal, lvl, val)
```

**Review by jsumners — APPROVED** [review]
LGTM

**Pull request merged**